app/templates/search/_categories.html

```diff
@@ -1,14 +1,47 @@
-<ul>
-  {% for category in categories %}
-    <li>
-      {% if category.link %}<a href="{{ category.link }}">{% endif %}{% if category.selected %}<strong>{% endif %}
-        {{ category.label }}
-      {% if category.selected %}</strong>{% endif %}{% if category.link %}</a>{% endif %}
-      {% if category.children and category.selected %}
-        {% with categories=category.children %}
-          {% include 'search/_categories.html' %}
-        {% endwith %}
+{%- macro top_level_link(text, url) -%}
+  <a href="{{ url }}" class="lot-filters__top-level-link"><span class="visually-hidden">Back to </span>{{ text }}</a>
+{%- endmacro -%}
+
+{%- macro list_item__unselected(text, url) -%}
```

Comment:
So I don't particularly like jinja macros - where they're the only solution to a problem that avoids a lot of repetition, then they're fine. But in this case it's just for two nested elements, and the macro's only used the once, so I'd prefer the markup just to appear inline?